claims_hosp/delphi_claims_hosp/update_indicator.py

```diff
@@ -152,15 +152,18 @@ def update_indicator(self, input_filepath, outpath, logger):
         data_frame = self.geo_reindex(data)
 
         # handle if we need to adjust by weekday
-        wd_params = Weekday.get_params(
-            data_frame,
-            "den",
-            ["num"],
-            Config.DATE_COL,
-            [1, 1e5],
-            logger,
-        ) if self.weekday else None
-
+        wd_params = (
+            Weekday.get_params_legacy(
+                data_frame,
+                "den",
+                ["num"],
+                Config.DATE_COL,
+                [1, 1e5],
+                logger,
+            )
+            if self.weekday
+            else None
+        )
```

Comment:
the linter is a cruel mistress!  id argue that my suggestion has a better style, even if the previous format passed, but the existing code is a fossilized remnant so do what thou wilt... 
```suggestion
        if weekday:
            wd_params = Weekday.get_params_legacy(
                data_frame,
                "den",
                ["num"],
                Config.DATE_COL,
                [1, 1e5],
                logger,
            )
        else:
            wd_params = None
```